src/layouts/MainLayout.vue

```diff
@@ -224,3 +224,10 @@ export default {
   margin-right: 0;
 }
 </style>
+
+<style global>
```

Comment:
Hi, please avoid declaring global styles. You can add this to the existing style. Thank you.

Addition: You can add a specific id to the nav component for this and target that.

Reply:
Hi i tried the solution you suggested but it didn't work, here's the code I used: 
```
---
  q-drawer(
    v-if="isMobile"
    v-model="drawerLeft"
    side="left"
    class="bg-primary fixed-aside"
    bordered
    :width="300"
    :breakpoint="500"
  )
---
</style scoped>
.fixed-aside{
  position: fixed !important;
  height: 100% !important;
}

.q-drawer{
  position: fixed !important;
  height: 100% !important;
}

aside{
  position: fixed !important;
  height: 100% !important;
}
</style>
```

## behavior 
- I noticed that when I add the fixed-aside class to the q-drawer element, it only applies the styles to the child element of the q-drawer component, not the q-drawer component itself. Additionally, the q-drawer component has its own default position of absolute.
- Do you have any suggestions? I'm not familiar with Quasar.

![image](https://user-images.githubusercontent.com/91131474/235012102-b1adc69f-6790-4a09-b762-5dacaebc9cb4.png)

Reply:
Hi i tried the solution you suggested but it didn't work, here's the code I used: 
```
---
  q-drawer(
    v-if="isMobile"
    v-model="drawerLeft"
    side="left"
    class="bg-primary fixed-aside"
    bordered
    :width="300"
    :breakpoint="500"
  )
---
</style scoped>
.fixed-aside{
  position: fixed !important;
  height: 100% !important;
}

.q-drawer{
  position: fixed !important;
  height: 100% !important;
}

aside{
  position: fixed !important;
  height: 100% !important;
}
</style>
```

## behavior 
- I noticed that when I add the fixed-aside class to the q-drawer element, it only applies the styles to the child element of the q-drawer component, not the q-drawer component itself. Additionally, the q-drawer component has its own default position of absolute.
-  I tried adding a unique ID to the q-drawer component, but unfortunately, it didn't work. 
- I also did some research on how to target the parent element of the q-drawer component, but couldn't find a solution. 

Do you have any other suggestions on how to resolve this issue?

![image](https://user-images.githubusercontent.com/91131474/235012102-b1adc69f-6790-4a09-b762-5dacaebc9cb4.png)

Reply:
Do you have any other suggestions on how to resolve this issue, or do you think we should close this PR for now? 

Thank you.

Reply:
Please put this on hold for now I will review and test the solution. I am also not sure why it doesn't work at the moment. Thank you